app/models/service_template_container_template.rb

```diff
@@ -0,0 +1,94 @@
+class ServiceTemplateContainerTemplate < ServiceTemplateGeneric
+  def self.default_provisioning_entry_point(_service_type)
+    '/Service/Generic/StateMachines/GenericLifecycle/provision'
+  end
+
+  # create ServiceTemplate and supporting ServiceResources and ResourceActions
+  # options
+  #   :name
+  #   :description
+  #   :service_template_catalog_id
+  #   :config_info
+  #     :provision
+  #       :dialog_id (or)
+  #       :new_dialog_name
+  #       :container_project_id
+  #       :new_project_name
```

Comment:
@bzwei That command is [here](https://github.com/ManageIQ/manageiq-providers-openshift/blob/10ff45acec1a7a700790997fd31c3bb7b64a0fc3/app/models/manageiq/providers/openshift/container_manager.rb#L23).